tests/10-data_model.js

```diff
@@ -0,0 +1,496 @@
+/*!
+ * Copyright (c) 2022-2023 Digital Bazaar, Inc. All rights reserved.
+ */
+// import * as sl from '@digitalbazaar/vc-status-list';
+import {addPerTestMetadata, decodeSl, getSlc, setupMatrix} from './helpers.js';
+import {testCredential, testSlCredential} from './assertions.js';
+import assert from 'node:assert/strict';
+import chai from 'chai';
+import {createRequire} from 'module';
+import {filterByTag} from 'vc-test-suite-implementations';
+import {TestEndpoints} from './TestEndpoints.js';
+
+const should = chai.should();
+
+const require = createRequire(import.meta.url);
+
+const tag = 'BitstringStatusList';
+const {match} = filterByTag({tags: [tag]});
+
+describe('Data Model: BitstringStatusList Entry', function() {
+  setupMatrix.call(this, match);
```

Comment:
this is not related to this PR, but this should be a helper function in the reporter which is exported instead of being a piece of copy and paste code.

Reply:
Good idea, the reporter should include 2 setup functions, one for conformance test and one for interop test